src/styles/main.scss

```diff
@@ -1,3 +1,8 @@
-body {
-  margin: 0;
-}
+@import "./utils/normalize.scss";
+@import "./utils/variables.scss";
+@import "./utils/functions.scss";
+@import "./blocks/catalog.scss";
+@import "./blocks/header.scss";
+@import "./blocks/nav.scss";
+@import "./blocks/card.scss";
+@import "./blocks/stars.scss";
```

Comment:
for better readability, you can separate utils styles and blocks styles with an empty line